File: programmers/lv0/코딩기초트레이닝/출력과연산/문자열겹쳐쓰기/Solution.go
package main

//func solution(my_string string, overwrite_string string, s int) string {
//	var result string = ""
//	for i := 0; i < s; i++ {
//		result += string(my_string[i])
//	}
//	result += overwrite_string
//	for i := len(result); i < len(my_string); i++ {
//		result += string(my_string[i])
//	}
//	return result
//}

func solution(my_string string, overwrite_string string, s int) string {
	return my_string[:s] + overwrite_string + my_string[s+len(overwrite_string):]
}

File: programmers/lv0/코딩기초트레이닝/출력과연산/문자열겹쳐쓰기/Solution.java
package lv0.코딩기초트레이닝.출력과연산.문자열겹쳐쓰기;

class Solution {
    public String solution(String my_string, String overwrite_string, int s) {
        return my_string.substring(0, s) +
                overwrite_string +
                my_string.substring(s + overwrite_string.length());
    }
}
